.github/workflows/elixir.yml

```diff
@@ -18,14 +18,14 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: '1.14'
-              otp: '23.3'
-          - pair:
-              elixir: '1.15'
+              elixir: '1.16'
```

Comment:
Elixir 1.18 came out rather recently. New versions are released roughly every 6 months, and v1.15 is only a year and a half old. IMO the minimum supported version should be at least Elixir v1.15 and OTP 24.3.